### PR TITLE
Update instructions for the word count exercise.

### PR DESCRIPTION
**Pull request by sateeshkumarb**
Closes https://github.com/exercism/rust/issues/1265

### CODE REVIEW

File: exercises/practice/word-count/.docs/instructions.md [hunk 1]
```diff
@@ -13,9 +13,10 @@ When counting words you can assume the following rules:
 1. The count is _case insensitive_ (ie "You", "you", and "YOU" are 3 uses of the same word)
 2. The count is _unordered_; the tests will ignore how words and counts are ordered
 3. Other than the apostrophe in a _contraction_ all forms of _punctuation_ are ignored
-4. The words can be separated by _any_ form of whitespace (ie "\t", "\n", " ")
+4. The words can be separated by _any_ form of whitespace (i.e. "\t", "\n", " "), or
+   punctutaion (except for single apostrophe in the contraction which is part of the word).
```
Comment (coriolinus) [inline exercises/practice/word-count/.docs/instructions.md:17]:
```suggestion
   external punctuation.
```

- fixed typo 
- phrasing: "external punctuation" seems clearer than "any punctuation except for single apostrophe". It's also more correct: given the input "He attended the barn-raising.", I'd expect the last word to be parsed as a single word "barn-raising", not as the two words "barn" and "raising".

Comment (sateeshkumarb) [reply exercises/practice/word-count/.docs/instructions.md]: Updated.

**Review by coriolinus — APPROVED** [review]
Looks great, thanks!

File: exercises/practice/word-count/.docs/instructions.md [hunk 2]
```diff
@@ -4,18 +4,19 @@ Given a phrase, count the occurrences of each _word_ in that phrase.
...
-1. The count is _case insensitive_ (ie "You", "you", and "YOU" are 3 uses of the same word)
+1. The count is _case insensitive_ (i.e. "You", "you", and "YOU" are 3 uses of the same word)
 2. The count is _unordered_; the tests will ignore how words and counts are ordered
 3. Other than the apostrophe in a _contraction_ all forms of _punctuation_ are ignored
-4. The words can be separated by _any_ form of whitespace (ie "\t", "\n", " ")
+4. The words can be separated by _any_ form of whitespace (i.e. "\t", "\n", " "), or
+   external punctuation.
 
-For example, for the phrase `"That's the password: 'PASSWORD 123'!", cried the Special Agent.\nSo I fled.` the count would be:
+For example, for the phrase `"That's the password: 'PASSWORD 123'!", cried the Special Agent.\nSo I fled."` the count would be:
```
Comment (kotp) [inline exercises/practice/word-count/.docs/instructions.md:19]: There are 3 `"` and this is not correct.  The last one is the one that should not be there.

**Review by kotp — COMMENTED** [review]
An additional quote was added where it should not be.

File: exercises/practice/word-count/.docs/instructions.md (hunk 2 above)
Comment (sateeshkumarb) [reply exercises/practice/word-count/.docs/instructions.md:19]: My bad. I have corrected this in the PR https://github.com/exercism/rust/pull/1269/. Please check.

**Pull request merged**